tools/genwqe_memcopy.c

```diff
@@ -839,11 +839,11 @@ int main(int argc, char *argv[])
 				/* FIXME: this is not 100 % good code, i know the format_flag is bad */
 				if (pt->total_usec < 100000) {
 					kibs = ((pt->bytes_copied * 1000000) / 1024) / pt->total_usec;
-					VERBOSE1("%d KiB, in %ld usec, %ld KiB/sec", kib, pt->total_usec, kibs);
+					VERBOSE1("%d KiB, in %lld usec, %lu KiB/sec", kib, pt->total_usec, kibs);
```

Comment:
VERBOSE1("%d KiB, in %llu usec, %lu KiB/sec", kib, (long long)pt->total_usec, kibs);